--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -434,9 +434,6 @@ endif()
 # SQLCipher option
 if(sqlcipher)
     add_definitions(-DENABLE_SQLCIPHER)
-    set(LIBSQLITE_NAME SQLCipher)
-else()
-    set(LIBSQLITE_NAME SQLite3)
 endif()
 
 # add extra library path for MacOS and FreeBSD
@@ -446,12 +443,8 @@ if(EXTRAPATH)
     list(PREPEND CMAKE_PREFIX_PATH /usr/local/opt/sqlitefts5/lib)
 endif()
 
-find_package(${LIBSQLITE_NAME})
-if (sqlcipher)
-    target_link_libraries(${PROJECT_NAME} SQLCipher::SQLCipher)
-else()
-    target_link_libraries(${PROJECT_NAME} SQLite::SQLite3)
-endif()
+find_package(SQLite3)
+target_link_libraries(${PROJECT_NAME} SQLite::SQLite3)
 
 if(MSVC)
     if(sqlcipher)
--- a/src/sqlite.h
+++ b/src/sqlite.h
@@ -6,14 +6,8 @@
     #ifndef SQLITE_HAS_CODEC
         #define SQLITE_HAS_CODEC
     #endif
-    #ifdef Q_OS_WIN32
-        #include <sqlite3.h>
-    #else
-        #include <sqlcipher/sqlite3.h>
-    #endif
-#else
-    #include <sqlite3.h>
 #endif
+#include <sqlite3.h>
 
 // For older versions of the SQLite library which do not have the SQLITE_DETERMINISTIC flag
 // yet (which introduced in this commit: https://www.sqlite.org/src/info/5716fc2341ddd8cf), we
